rmgpy/data/thermoTest.py

```diff
@@ -295,6 +295,20 @@ def testSpeciesThermoGenerationLibrary(self):
         self.assertTrue(arom.isIsomorphic(spec.molecule[0]))  # The aromatic structure should now be the first one
         self.assertTrue('library' in thermo.comment, 'Thermo not found from library, test purpose not fulfilled.')
 
+    def testThermoEstimationNotAffectDatabase(self):
+
+        poly_root = self.database.groups['polycyclic'].entries['PolycyclicRing']
+        previous_enthalpy = poly_root.data.getEnthalpy(298)/4184.0
+        smiles = 'C1C2CC1C=CC=C2'
+        spec = Species().fromSMILES(smiles)
+        spec.generateResonanceIsomers()
+
+        thermo_gav = self.database.getThermoDataFromGroups(spec)
```

Comment:
Should you add an assertion that the generated thermo data actually hit the `PolycyclicRing` group? Or is that always guaranteed?

Reply:
will add that.